### DISCUSSION TITLE
deprecated material

### DISCUSSION THREAD

**Discussion by ghost** [cyrilletuzi/vscode-angular-schematics#926] [Q&A] [answered]
During the installation process of the 'Angular Schematisc' extension (v6.9.0), Visual Studio Code generated the following warnings due to deprecated material as of the current date:

 npm install --save-dev @compodoc/compodoc
npm WARN deprecated @babel/plugin-proposal-private-methods@7.18.6: This proposal has been merged to the ECMAScript standard and thus this plugin is no longer maintained. Please use @babel/plugin-transform-private-methods instead.
npm WARN deprecated sourcemap-codec@1.4.8: Please use @jridgewell/sourcemap-codec instead
npm WARN deprecated vis@4.21.0-EOL: Please consider using https://github.com/visjs

I preferred to report the situation due to the risk that in the near future such elements might no longer be accessible

**Comment by cyrilletuzi** [comment, marked as answer]
Seems totally unrelated to this extension, which does not install anything via npm. Maybe VS Code has also updated another extension while installing the Angular Schematics one.
